applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -62,16 +45,36 @@ def __init__(self, Model, settings):
                 node.Set(KratosMultiphysics.SOLID)
 
     def ExecuteInitialize(self):
+        self.__FindWakeElements()
+
+    def __FindWakeElements(self):
+        self.__ReadWakeDirection()
         # Save the trailing edge for further computations
-        self.SaveTrailingEdgeNode()
+        self.__SaveTrailingEdgeNode()
```

Comment:
Why name mangling in all these methods?

Reply:
This is intended to follow the style guide, which was pointed out in:
https://github.com/KratosMultiphysics/Kratos/pull/4642#discussion_r274928255

Reply:
The point is that in Python doing __MyMethod mangles the name, which is not private at all... In any case I wouldn't block the PR because of this.

Reply:
This was the agreement for "private" methods, which was discussed long ago
Ofc you could still access them but it gives a hint to the user to not do it...

Reply:
OK then! From now on I'll follow this convention.